docs/copiloting-service.md

```diff
@@ -0,0 +1,119 @@
+
+- explaining and demonstrating how OpenSAFELY works
+- weekly meetings to scope out achievable weekly objectives and tackle problems
+- support with the construction of the required patient cohort
+- support with the identification of clinical/demographic variables of interest
+- support with identifying or building codelists on [opencodelists.org](https://www.opencodelists.org/)
+- help with navigating [the OpenSAFELY documentation](https://docs.opensafely.org) to find necessary documentation and policies
+- help with navigating [the OpenSAFELY GitHub repository](https://github.com/opensafely) to find helpful examples
+- advice on implementing quality assurance steps
+- advice on disclosure control and redaction as carried out by our output checking service
+- help with troubleshooting errors when running the study locally or on the server
+- coaching, advice and feedback with regards to best practice on GitHub
+- communicating with the OpenSAFELY tech team on the pilot's behalf with regards to platform development
+
+Please note that copilots will not perform code review of any analysis scripts or review any codelists that are developed. This is because this expertise lies with the pilot and their wider research team; pilots will be advised to identify someone within their own organisation who is willing to review this component of the project.
+
+## What happens after the copiloting period is over (the 'post copiloting stage')? 
+
+It is not expected that the research project will be complete at the end of the active copiloting period, rather that the pilot will have acquired the relevant experience to complete the project independently. All community resources (the [discussion forum](https://github.com/opensafely/documentation/discussions) and the `#opensafely-users` Slack channel) will remain accessible to the pilots but regular, dedicated 1:1 support for study implementation will end. The copilot will remain involved in the project over the long term in terms of output checking and manuscript review. 
+
+All OpenSAFELY outputs require approval from NHS England before they can be disseminated any wider than the pilot's research group. This includes academic manuscripts (pre-printed or peer-reviewed), conference abstracts and presentations, internal reports and masters/PhD theses. Our IG team will handle the NHS England approval process; for more information about how to request NHS England approval and what to include in your publication text please see [this section of our documentation](https://www.opensafely.org/policies-for-researchers/).
+
+As with all OpenSAFELY projects, once a copiloted project is finished, published or it reaches its 1 year anniversary, the associated GitHub repository (and Jobserver outputs) need to be made public (see [this section of our documentation](https://docs.opensafely.org/publishing-repo/) for more information).
+
+## What is our authorship policy?
```

Comment:
Is this anywhere else?

i.e. we should not be maintaining two versions - possibly its on the website under "policies for researchers"

Reply:
There is the following text on the Policies for Researchers page:

> Upon publication of any associated papers, presentations, etc (and in any case within 12 months of first code execution) you must publish your Github repository.

...so pretty limited information there. However, I take your point and maybe we should remove this altogether as people reading this won't need to know those details anyway.

Reply:
Can you find the draft please and ask for one of the research teams to add to their project board to get it finished over the next few weeks/months?

Reply:
Oh its here https://cd538e02.opensafely-docs.pages.dev/copiloting-service/#what-is-our-authorship-policy

Can you maybe start discussion on slack about where the best place is to host? e.g. website (policies for researchers) versus docs as per link above

Reply:
[Conversation prompted](https://bennettoxford.slack.com/archives/C31D62X5X/p1673978938647819).

Reply:
I have added an issue to remind us to revisit this.
https://github.com/opensafely/documentation/issues/1103